src/flownet/config_parser/_config_parser.py

```diff
@@ -303,6 +377,9 @@ def create_schema(_to_abs_path) -> Dict:
                     },
                     "bulkvolume_mult": {
                         MK.Type: types.NamedDict,
+                        MK.Description: "Description of bulk volume multiplier "
```

Comment:
Should this describe a bit more what this actually does? Or should that be a separate part of the docs?